src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -506,52 +510,51 @@ def launch_fluent(
     start_transcript : bool, optional
         Whether to start streaming the Fluent transcript in the client. The
         default is ``True``. You can stop and start the streaming of the
-        Fluent transcript subsequently via method calls on the session object.
+        Fluent transcript subsequently via the method calls, ``start_transcript()``
+        and ``stop_transcript()`` on the session object.
     show_gui : bool, optional
-        Whether to display the Fluent GUI when ``start_instance``
-        is set to ''True``. The default is ``None`` so that explicit
-        ``False`` settings can be detected. This is because you can use
-        also use the environment variable ``PYFLUENT_SHOW_SERVER_GUI=<0 or 1>``
-        to set this parameter. The ``show-gui`` parameter overrides the
-        PYFLUENT_SHOW_SERVER_GUI environment variable. For example, if
-        PYFLUENT_SHOW_SERVER_GUI is set to ``1`` and the ``show-gui``
-        parameter is set to ``False``, the GUI is hidden.
+        Whether to display the Fluent GUI, only when ``start_instance``
+        is set to ``True``. The default is ``None``, which does not
+        cause the GUI to be shown. If a value of ``False`` is
+        not explicitly provided, the GUI will also be shown if
+        the environment variable ``PYFLUENT_SHOW_SERVER_GUI`` is set to 1.
     case_filepath : str, optional
-        If provided, reads a fluent case file and sets the required settings
-        in the fluent session
+        If provided, the case file at ``case_filepath`` is read into the Fluent session.
     case_data_filepath : str, optional
-        If provided, reads a fluent case and data file and sets the required settings
-        in the fluent session
+        If provided, the case and data files at ``case_data_filepath`` are read into the Fluent session.
     lightweight_mode: bool, optional
-        Whether to use light io mode for reading case via ``case_filepath`` parameter.
-        This parameter is used only when ``case_filepath`` is provided. The default is
-        ``False``.
+        Whether to read only the lightweight settings from any specified Fluent case file, ignoring the mesh.
+        This parameter is used only when ``case_filepath`` is provided. The default is ``False``.
     mode : str, optional
         Launch mode of Fluent to point to a specific session type.
         The default value is ``None``. Options are ``"meshing"``,
         ``"pure-meshing"`` and ``"solver"``.
     server_info_filepath: str
-        Path to server-info file written out by Fluent server. The default is ``None``.
+        Path to server-info file written out by Fluent server. The default is
+        ``None``. ``server_info_filepath`` can be specified either if ``start_instance``
+        is ``True``, where PyFluent will launch and then direct Fluent to write the file
+        at that path, or if ``start_instance`` is ``False``, where PyFluent will use the
```

Comment:
> then direct Fluent to write the file at that path

@seanpearsonuk I don't think this is supported

Reply:
@mkundu1 I didn't feel that the current version of the docstring was explicit enough as to the purpose of the argument (it has caused some confusion amongst users) so I had a quick look at the code and surmised what's written above. It's possible that what I've written is not accurate.

Reply:
@mkundu1 I tried this out - I will update this docstring